packages/main/src/AvatarGroup.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="ui5-avatar-group-root" dir="{{effectiveDir}}">
+	<div 
```

Comment:
Do we need these 2 divs for a specific reason? One display: flex and inside display: inline-flex?
If this isn't a hack for something, please use one div only.

Reply:
2 DIVs are needed because the first one is used to attach the resize listener on it, its width is used to calculate how many items should overflow. The second one is inline-flex so its width will be eqaul to the width of its content and focus styles will be correctly set (when type="Group").